libs/extensions/angular/project.json

```diff
@@ -86,8 +86,8 @@
       "builder": "@storybook/angular:build-storybook",
       "options": {
         "configDir": "libs/extensions/angular/.storybook",
-        "browserTarget": "extensions-angular:build",
         "outputDir": "dist/storybook/extensions/angular",
+        "enableProdMode": false,
```

Comment:
Workaround for [this](https://github.com/storybookjs/storybook/issues/14828#) storybook issue